Code/GraphMol/FileParsers/MolSGroupParsing.cpp

```diff
@@ -105,6 +113,11 @@ void ParseSGroupV2000VectorDataLine(IDX_TO_SGROUP_MAP &sGroupMap, RWMol *mol,
 
   unsigned int pos = 6;
   unsigned int sgIdx = ParseSGroupIntField(text, line, pos);
+  if (sGroupMap.find(sgIdx) == sGroupMap.end()) {
+    BOOST_LOG(rdWarningLog) << "SGroup " << sgIdx << " referenced on line "
```

Comment:
Perhaps "on line: " to be consistent with other error reporting, not that we  really are anyway :)